@navikt/core/react/src/tabs/Tabs.tsx

```diff
@@ -0,0 +1,100 @@
+import cl from "classnames";
+import React, { createContext, forwardRef, HTMLAttributes } from "react";
+import * as RadixTabs from "@radix-ui/react-tabs";
+import Tab, { TabType } from "./Tab";
+import List, { ListType } from "./TabList";
+import Panel, { PanelType } from "./TabPanel";
+
+export interface TabsProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, "onChange" | "dir"> {
+  /**
+   * Tabs elements
+   */
+  children: React.ReactNode;
+  /**
+   * Changes padding and font-size
+   * @default "medium"
+   */
+  size?: "medium" | "small";
+  /**
+   * onChange
+   */
+  onChange?: (value: string) => void;
+  /**
+   * Controlled selected value
+   */
+  value?: string;
+  /**
+   * If not controlled, a default-value needs to be set
+   */
+  defaultValue?: string;
+  /**
+   * Automatically activates tab on fokus/navigation
+   * @default false
+   */
+  autoSwitch?: boolean;
+  /**
+   * Makes icons larger for better visibility
+   */
+  iconOnly?: boolean;
```

Comment:
Jeg synes material-ui sitt API for med separate `icon`- og `label`-props var ganske nice her: https://mui.com/components/tabs/#icon-position. Der har de også en egen prop for posisjon til ikonet.